src/server.js

```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+
+const port = process.argv.PORT || 5000;
+
+const server = http.createServer();
+
+server.listen(port, () => {
+  // eslint-disable-next-line no-console
+  console.log(`server is listining to the port: ${port}`);
```

Comment:
This console line need to be changed,
try to put the localhost link in the console and correct `listining`.
```Visit our server at localhost http://localhost:${port}```